### PR TITLE
Improve documentation for contributors

### PR DESCRIPTION
**Pull request by tijmenb**
See commits for details.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Contributing to this project
+
+Everybody is encouraged to contribute to this project. If you're part of the [alphagov GitHub organisation](https://www.github.com/alphagov) you probably have write access to this repo. If you don't, ask a GitHub admin or #tech-docs-format on Slack.
```
Comment (arnau) [inline CONTRIBUTING.md]: Who is "everybody" here? anyone with a Github account or anyone part of Alphagov?

File: README.md [hunk 2]
```diff
@@ -40,6 +43,14 @@ To run the tests on your machine:
 
 Or, on the command line, run `bundle exec rake jasmine:ci`.
 
+## Releasing new versions
+
+At the moment this gem has to be released manually. There's a [story on the Reliability Engineering backlog](https://trello.com/c/gRQ8OnBl) to automate this.
+
+To release a new version, create a new Pull Request that updates [version.rb](lib/govuk_tech_docs/version.rb) and [CHANGELOG.md](CHANGELOG.md). Don't mix this in with other changes - this makes it easier to find out what what released when. See [this PR to release a new version as an example](https://github.com/alphagov/tech-docs-gem/pull/15).
```
Comment (barrucadu) [inline README.md]: > find out what what released when

Should that be "find out what released when"?

File: CONTRIBUTING.md (hunk 1 above)
Comment (tijmenb) [reply CONTRIBUTING.md]: Updated in https://github.com/alphagov/tech-docs-gem/pull/21/commits/3937eb1a08e7a11c863a9b388ec9a720db6a4a0a

File: README.md (hunk 2 above)
Comment (tijmenb) [reply README.md]: Fixed in https://github.com/alphagov/tech-docs-gem/pull/21/commits/ac94924f1a3fc7ba30ea7297c295aacaf7d39a9a

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+## Contributing to this project
+
+Everybody who uses this project is encouraged to contribute to this project. If you're part of the [alphagov GitHub organisation](https://www.github.com/alphagov) you probably have write access to this repo. If you don't, ask a GitHub admin or #tech-docs-format on Slack.
+
+- Before adding your feature, check [the backlog if someone is already talking about it](https://github.com/alphagov/tech-docs-template/issues)
```
[the remark this answers is not in the document]
Comment (tijmenb) [reply CONTRIBUTING.md]: Updated!

**Pull request merged**